src/main/java/seedu/address/logic/commands/UnmarkTaskCommand.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.Task;
+import seedu.address.model.task.TaskList;
+
+/**
+ * Marks a task as incomplete in TaskHub.
+ */
+public class UnmarkTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "unmarkT";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Marks a task as incomplete in TaskHub.\n"
+            + "Parameters: " + PREFIX_PROJECT + "PROJECT_INDEX " + PREFIX_TASK + "TASK_INDEXES\n"
+            + "PROJECT_INDEX must be one positive integer and "
+            + "TASK_INDEXES must be one or more positive integers, separated by a space between each INDEX\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_PROJECT + "2 " + PREFIX_TASK + "1 2 4";
+
+    public static final String MESSAGE_TASKS_UNMARKED_SUCCESSFULLY = "%1$d task(s) marked as incomplete.";
```

Comment:
Similar to the above comment, perhaps you would like to add the project index in the successful message.